shared/metrics/src/main/java/io/pravega/shared/metrics/MetricsProvider.java

```diff
@@ -9,10 +9,18 @@
  */
 package io.pravega.shared.metrics;
 
+import com.codahale.metrics.Metric;
 import com.codahale.metrics.MetricRegistry;
+import com.google.common.annotations.VisibleForTesting;
 
 public class MetricsProvider {
-    static final MetricRegistry METRIC_REGISTRY = new MetricRegistry();
+
+    @VisibleForTesting
+    public static Metric getMetric(String name) {
```

Comment:
please move this below variable declaration and along with other  static methods

Reply:
done.